web-frontend/src/pages/CreateAccount/StepThree.tsx

```diff
@@ -0,0 +1,36 @@
+import { Stepper } from '@massalabs/react-ui-kit/src/components/Stepper/Stepper';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+import { FiArrowRight } from 'react-icons/fi';
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+
+export default function StepThree() {
+  return (
+    <LandingPage>
+      <div
+        className={`flex flex-col justify-center items-center align-center h-screen`}
+      >
+        <div className="flex flex-col justify-center items-center w-fit h-fit max-w-sm">
+          <div className=" w-full max-w-xs mb-6">
+            <Stepper step={2} steps={['Username', 'Password', 'Back-up']} />
```

Comment:
Again here! Backup is one word!
```suggestion
            <Stepper step={2} steps={['Username', 'Password', 'Backup']} />
```

Reply:
![Screenshot from 2023-05-16 10-31-15](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/0f6a7fe9-9e6b-42d7-83c8-b32aeb1f2bf4)

back-up is seperated in designs ://

Reply:
see pr [here](https://github.com/massalabs/thyra-plugin-wallet/pull/202)